### DISCUSSION TITLE
Question about formatting the "About Page"

### DISCUSSION THREAD

**Discussion by davidpassmore** [quarto-dev/quarto-cli#11070] [Q&A]
### Description

I created an "About" page in RStudio using the "broadside" template demonstrated at https://quarto.org/docs/websites/website-about.html#templates. This template produces 2 columns, one containing an image and the other containing text I code as partial page content (https://quarto.org/docs/websites/website-about.html#partial-page-content).

Can I code content that is full-page width below the partial page content? I have not seen this matter addressed in other discussions.

Perhaps it would be easier to code my own "About" page without using a template?

Thank you,
David

**Comment by mcanouil** [comment]
> Can I code content that is full-page width below the partial page content

I don't understand, that's exactly what the link to the documentation you put is about.

Setting ID and using a fenced div allows to put any arbitrary content formatted as you wish around it, as illustrated in the documentation page you've linked.
